CFC_WebApp/clients/socialmedia/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "socialmedia",
+  "start_date": "2014/10/13",
+  "end_date": "2015/06/01",
+  "key": "dc23ad22-bc78-4fd6-9adf-ca2b04505916",
```

Comment:
This was an arbitrary key I was using to figure out what was going on with the clients. I’ll change it back to whatever the dummy key was.

Sid Feygin
sid.feygin@gmail.com



> On Jun 2, 2015, at 5:00 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/clients/socialmedia/settings.json <https://github.com/e-mission/e-mission-server/pull/133#discussion_r31583553>:
> 
> > @@ -0,0 +1,11 @@
> > +{
> > +  "name": "socialmedia",
> > +  "start_date": "2014/10/13",
> > +  "end_date": "2015/06/01",
> > +  "key": "dc23ad22-bc78-4fd6-9adf-ca2b04505916",
> You generally don't want the key to be checked in. It should be generated on each server.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e-mission/e-mission-server/pull/133/files#r31583553>.
>